Mistica/Source/Components/Callout/Internals/CalloutButtons.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  CalloutButtons.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import UIKit
+
+/// CardButtons is a custom view which contains the buttons  that belong for the callouts
+class CalloutButtons: UIStackView {
+    let primaryButton = Button()
+    let secondaryButton = Button()
+    let linkButton = Button()
+    private let dummyView = SpacerView(axis: .vertical, amount: 1)
+
+    private var primaryActionHandler: (() -> Void)?
+    private var secondaryActionHandler: (() -> Void)?
+    private var linkActionHandler: (() -> Void)?
+
+    override var accessibilityElements: [Any]? {
+        get {
+            [
+                primaryButton.superview == nil ? nil : primaryButton,
+                secondaryButton.superview == nil ? nil : secondaryButton,
+                linkButton.superview == nil ? nil : linkButton
+            ].compactMap { $0 }
+        }
+        set {}
+    }
+
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        commomInit()
+    }
+
+    required init(coder: NSCoder) {
+        super.init(coder: coder)
+        commomInit()
+    }
+}
+
+// MARK: Internal
+
+extension CalloutButtons {
+    func configureButtons(primaryButton: CalloutButton?, secondaryButton: CalloutButton?, linkButton: CalloutLinkButton?) {
+        if let primaryAction = primaryButton {
+            self.primaryButton.title = primaryAction.title
+            self.primaryButton.loadingTitle = primaryAction.loadingTitle
+            primaryActionHandler = primaryAction.tapHandler
```

Comment:
We prefer name overloads:

```suggestion
        if let primaryButton = primaryButton {
            self.primaryButton.title = primaryButton.title
            self.primaryButton.loadingTitle = primaryButton.loadingTitle
            primaryActionHandler = primaryButton.tapHandler
```